docs/source/markdown/podman-system-check.1.md

```diff
@@ -34,6 +34,12 @@ attempts to pull images, and should be treated as though they are damaged.
 Skip checks which are known to be time-consuming.  This will prevent some types
 of errors from being detected.
 
+#### **--quicker**, **-Q**
+
+Like --quick but also skip checking layer digests. This is even quicker but may
+fail detecting whether a file is missing from the image or its content has
+changed.
```

Comment:
I don't like --quicker as name, and I don't like conflicting short names that just differ between upper and lower case.
Logically what happens if someone can add an even quicker option? Then the cli would get even worse, etc...

There is also no conflict handling between --quick and --quicker.

If all it does is skip the digest check then maybe given it a name saying so would be better. Then it could be use in addition to quick or without it,